backend/celery_worker.py

```diff
@@ -319,4 +431,8 @@ def check_if_is_premium_user():
         "task": "check_if_is_premium_user",
         "schedule": crontab(minute="*/1", hour="*"),
     },
+    "process_assistant": {
```

Comment:
Does the assistant need to run every minute ?

Reply:
no